src/open_inwoner/webcomponents/components/ExternalLinks.ts

```diff
@@ -0,0 +1,88 @@
+import { LitElement, html, css } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+import 'src/open_inwoner/static/bundles/open_inwoner-css.css'
```

Comment:
@jiromaykin If I [read the docs correctly](https://vite.dev/guide/features#css-pre-processors), something like this should work (will need `npm add -D sass-embedded`):

```suggestion
import styles from 'src/open_inwoner/scss/components/ExternalLink/ExternalLink.scss?inline'
```

Then in the component attributes:

`static styles = css(styles)` (or perhaps `unsafeCSS`)

Reply:
i've moved the SCSS to the components own directory + i had to add more than 1 SCSS file due to Material Icons and shadow-DOM workings, so it becomes styles array.